package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@hackerrank/firepad",
   "description": "Collaborative text editing powered by Firebase",
-  "version": "0.4.1-beta",
+  "version": "0.5.1-alpha",
```

Comment:
can we use 0.7.0-beta? because the 0.5.x, and 0.6.x are already used from the rolled-out branches and from our previous PR

Reply:
Fixed.